internal/engine/upper.go

```diff
@@ -54,7 +54,6 @@ type Upper struct {
 	podWatcherMaker     PodWatcherMaker
 	serviceWatcherMaker ServiceWatcherMaker
 	k8s                 k8s.Client
-	browserMode         BrowserMode
```

Comment:
do you mean to be removing this code as part of this PR?

Reply:
yep, this is all dead anyway, and is slowly getting removed. i didn't want to change the demo.Script to wire up BrowserMode